README.md

```diff
@@ -19,78 +26,71 @@ To install from source, follow the instructions in the [documentation](https://q
+This simple example makes an entangled state, also called a [Bell state](https://qiskit.org/textbook/ch-gates/multiple-qubits-entangled-states.html#3.2-Entangled-States-).
+
+Once you've made your first quantum circuit, you can then simulate this circuit.
+To do this, first we need to compile your circuit for the target backend we're going to run
+on. In this case we are leveraging the built-in `BasicAer` simulator. However, this
+simulator is primarily for testing and is quite basic and very limited (as the name
+implies). You should consider using [`qiskit-aer`](https://github.com/Qiskit/qiskit-aer/)
+for any real simulation work.
 
 ```
-$ python
+from qiskit import transpile
+from qiskit.providers.basicaer import QasmSimulatorPy
+backend_sim = QasmSimulatorPy()
+transpiled_qc = transpile(qc, backend_sim)
 ```
 
-```python
->>> from qiskit import QuantumCircuit, transpile
->>> from qiskit.providers.basicaer import QasmSimulatorPy
->>> qc = QuantumCircuit(2, 2)
->>> qc.h(0)
->>> qc.cx(0, 1)
->>> qc.measure([0,1], [0,1])
->>> backend_sim = QasmSimulatorPy()
->>> transpiled_qc = transpile(qc, backend_sim)
->>> result = backend_sim.run(transpiled_qc).result()
->>> print(result.get_counts(qc))
+After compiling the circuit we can then run this on the ``backend`` object with:
+
+```
+result = backend_sim.run(transpiled_qc).result()
+print(result.get_counts(qc))
 ```
 
-In this case, the output will be:
+The output from this will be something like:
 
 ```python
 {'00': 513, '11': 511}
 ```
 
-The basic and very limited Qiskit build-in `BasicAer` simulator is probably all you need initially. However you should consider [`qiskit-aer`](https://github.com/Qiskit/qiskit-aer/) for any real simulation work.
-
-### Executing your code on a real quantum chip
-
-You can also use Qiskit to execute your code on a
-**real quantum chip**.
-Qiskit enables you to run code on real quantum computers. To to do this it provides an abstraction
-layer that lets hardware vendors build packages that provde an interface to their hardware from Qiskit.
-Using these ``providers`` you can run any Qiskit code against a real quantum computer.
-
-For example, to use Qiskit with IBM Quantum's cloud quantum computing service:
-
-#### Configure your IBM Quantum credentials
-
-1. Create an _[IBM Quantum](https://quantum-computing.ibm.com)_ account if you haven't already done so.
-
-2. Get an API token from the IBM Quantum website under _API Token_.  
-<a href="https://user-images.githubusercontent.com/766693/189518233-fa463e4d-08b8-4a0f-b61c-5d5e978acd8b.png"><img src="https://user-images.githubusercontent.com/766693/189518233-fa463e4d-08b8-4a0f-b61c-5d5e978acd8b.png" width="80%"/></a>
+For further examples of using Qiskit you can look at the example scripts in **examples/python**. You can start with
+[using_qiskit_terra_level_0.py](examples/python/using_qiskit_terra_level_0.py) and working up in the levels. Also
```

Comment:
Do we wont to keep the example folder?

Reply:
I left it in for now because I think it's probably worth calling out the examples while we have them. I think we can maybe debate removing the examples and when/if we do that we can drop this at the same time. But I agree they're probably not the most up-to-date and I would be in favor of removing them (especially as we've already removed the ibmq specific ones recently).

Reply:
Sounds good